### ISSUE TITLE
Images in protective scrolls: drawings or miniatures?

### ISSUE THREAD

**Issue by DariaElagina** [BetaMasaheft/Documentation#1794]
Dear all,

in this issue, I would like to clarify our definition of images found in protective scrolls. Commonly, they are represented by images of angels (with swords), ṭalsams, or sometimes by images of demons. We have two possibilities of classifying them, either as drawings or as miniatures. However, the definitions for these terms in the project do not correspond accurately to the images found in protective scrolls. 
Our definition for a miniature reads: "An independent illustration in colour that occupies the full-page or part of the equivalent space of one or more columns of text." 
Images in protective scrolls are neither independent (they always suppose to build a specific sequence with texts) nor designed as illustrations.
Our definition for a drawing: "A monochrome or bichrome image that is not painted." 
Images in protective scroll might vary in execution. The same image in one manuscript might be colored, and in another one not. 
I would be happy if we could just document which is our conventional way to define images in protective scrolls. I personally always tend to describe them as "drawings". What do you think, @thea-m , @DenisNosnitsin1970 , @eu-genia , @vitagrazia81 , @SophiaD-M , @gnizla @MassimoVilla  ?

**Comment by DenisNosnitsin1970** [comment]
There are a few debatable points in the definitions, or just things that can be questioned / clarified / made more understandable again, at least in my perception. The definition of "Drawing" is one of them (there is also a vague boundary between miniature that is not finished and a true drawing). "Independent" is used in a  special sence as most of the miniatures in manuscripts do depend upon texts because they illustrate them in one way or another. But how to describe talismanic images - are they miniatures? Are their any examples of catalogues distinguishing between them?

**Comment by DariaElagina** [comment]
You are right, some clarification would definitely be helpful. In my understanding "independent" means that they might or might not be there. The text does not lose its function or meaning if no miniatures are present, which might be a choice of taste or money. Whereas the presence of talsams, as far as I understand, is something more essential, that can not be reduced to a decorative or illustrative function. The basic difficulty for me in the definitions is the mixture of formality and functionality. In the case of a miniature, what comes as first: the fact that it should be an illustration or the fact that it should be colored? If the decisive factor is its illustrative character then magic images are not miniatures. If the decisive factor is the presence of color, they might be classified like that (though I am personally against this interpretation).

In the catalogue of Strelcyn (British Library), a very common "magical picture" is used. Once he uses "magical drawings" (p. 138), but since the manuscript is not digitized, there is no way to understand what the difference is. Describing other manuscripts, of no protective/magical nature, he uses "drawings" (for monochrome units) and "miniatures".
Mercier in "Art that Heals" uses quite often the term "talisman", but never a miniature. I will check also further catalogues once in the office.

**Comment by gnizla** [comment]
I agree with the points made by @DenisNosnitsin1970 there is not perfect distinction between a drawing and a miniature, but a distinction is needed and that will inevitably have to do with polychromy and colouring which are germane to a miniature and in antithesis with what a drawing is.

Personally, from an encoding point of view, I think it makes sense to treat the images on scrolls just like those in manuscripts: if they are monochrome, define them as drawings, if they are polychrome, define them as miniatures. If you want to add additional information you could then optionally create an art theme for each of those motifs that recurr on the scrolls and which are not attested in other manuscripts (e.g. Net of Solomon).

I can see why the term "indipendent" might cause some confusion, but a miniature is never indipendent from a text. If I recall correctly, indipendent was added to distinguish miniatures from onramental borders and frames in a discussion we had over a year ago.

**Comment by eu-genia** [comment]
I think the issue is not really drawing/miniature (I agree with Jacopo, would go by colouring), the confusion comes from the structure of the TEI msdesc which puts the text at the centre and decoration as an add-on and the fact that while most codices have  indeed the text as the main content, magic scrolls or leporellos have the images as part of the main content. But we have to use the structure we have.

**Comment by gnizla** [comment]
- I don't think that the size of an image, if I understood your intended meaning, is a valuable criteria for the type of distinction we are talking about (as interesting as it may be from a "scribal" perspective).

- When drawing on catalogues for pieces that are not digitized, especially older ones, I believe it is implicit that in everything one does one makes some judgment calls (and not just for images), I would opt for referring to the images as miniatures in the case of doubt, as ones with colour of some kind (at times just two colours) are far more common than ones without, but that's just my two cents!


Thanks!

**Comment by DariaElagina** [comment]
Thank you, @gnizla ! These are very helpful remarks! Do I understand you correctly that you would suggest treating bichrome images also as miniatures? The current definitions treat them as "drawings".

**Comment by gnizla** [comment]
Dear @DariaElagina Daria, thanks for query.

From my point of view, once you draw the distinction based on colouring, then yes you could include bichrome drawings in the "miniature" category. Of course, this is a type of image that sits right across the boundary of our discussion for the fact that they are produced with the inks that are used for scribing manuscripts rather than being coloured, so to speak. So, this could go either way.

If bichrome images have not been treated as miniatures in encoding done so far (and if that encoding is substantial in volume) then it may be easier to retain the current approach even if it generates some conceptual discrepancies. But, yes, purely from a logical standpoint, I would suggest categorizing them as miniatures as it creates a simpler and more clear approach towards dividing miniatures and drawings.

**Comment by DariaElagina** [comment]
Thank you, @gnizla ! I think we have now to decide which strategy we will follow. It would be nice to hear the opinion of the others. I have also a different question concerning the word "independent", if I understand correctly, this criterion should also be included in the definition of a drawing, right? 
Concerning this term you wrote as follows:
> 
> 
> I can see why the term "indipendent" might cause some confusion, but a miniature is never indipendent from a text so I don't think it is possible to draw a marked distinction between how images function on scrolls and manuscripts (which is not to say that there are no such differences). In any case, if I recall correctly, indipendent was added to distinguish miniatures from onramental borders and frames in a discussion we had over a year ago. Denis and Dorothea where in that conversation if I recall correctly.

Would it be possible to rather add the term "aniconic" to the definitions of frames, bands, and other ornamental elements? (as applied in the COMSt Handbook, p. 165)

**Comment by gnizla** [comment]
Just to clarify, @DariaElagina I believe I did not introduce the term "indipendent" so I am not advocating to retain it. 

Aniconic is a useful term, but some bands have little heads in them, so not always true.

Sure, good to hear other opinions!

**Comment by DenisNosnitsin1970** [comment]
Some points: if it is desirable to distinguish the images of scrolls as a special group, maybe the term "talismanic" maybe used, e.g. "talismanic miniature" (for all types, cp. the catalogues of V. Six) or "talismanic drawing" ("magical" is clear but perhaps can be found questionable by some people), they sometimes occur in the manuscripts as well. I would also say that if the drawing is made of one colour (e.g. black) or two colour outlines (black and red, it may be also yellow, rarely), and the areas inside a not covered with paints, it is not a miniature but a (bicrome) drawing. "Independent" is clear for me now, it is not so important but it does allow multiple interpretations, probably it should be avoided in the guidelines.

**Comment by eu-genia** [comment]
I was wondering - we might have instances of completely "independent" images though, especially if they were added as a separate layer. maybe the term could be reserved for such cases.

**Comment by DenisNosnitsin1970** [comment]
If you mean miniatures on rejected leaves, added into a volume purely by chance, yes, but just the word "independent" has for me a different semantics - I wonder if we can say that such a miniature is "independent" as the miniature is not agent and does not decide, in such a form it is rather "stray" as "rejected leaves" are. Also, whoever paints a miniature, usually has a purpose.

**Comment by DariaElagina** [comment]
Thank you all for the valuable input!
1. I would be against introducing a special category for talismanic images because, in fact, it might be even more confusing. Categorization of an image as a talismanic one is 1. not straightforward; 2. again adds another layer (functionality) to our definitions, which imho we should avoid. If one is completely sure that this is a talismanic image, it can be added into the description. I just want to document which is the best way to encode (define) them, since our definitions, as they are, are too restrictive in my opinion.
2. My intention is to formulate definitions in a way that would allow for the disambiguation of a miniature and a drawing. If not all miniatures are "independent" it can not be a decisive criterion. And if all miniatures are independent and this delimits them from decorative elements (frames and ornamental bands), the same should be applicable to drawings. Because we have agreed that the difference is in colouring. Alternatively, we can introduce a third category, which would be used for anything which is neither a drawing nor a miniature :) But if we keep two terms, the disambiguation should be clearly supported by definitions.
3. @DenisNosnitsin1970 , how would you then encode those images for which no information on colouring is available, as miniatures or as drawings? And could you, please, elaborate on the term "independent"?

**Comment by gnizla** [comment]
1. I would also stay away from "Talismanic" as a special category (which does not mean, as Daria says, that it cannot be used in description).
2. I find "indipendent" confusing even based on this conversation alone. I would likewise do away with bicrome drawing. I think two categories are enough and more would be confusing. To be clear I am not disagreeing with @DenisNosnitsin1970 and one could easily debate whether it is more appropriate to define images with two colours as drawings or miniatures without coming to a conclusion. There is no right or wrong answer to this question, but, I would suggest categorizing these as miniatures to avoid any possible ambiguities: i.e. 1 colur = drawings 2+ colours = miniature.

**Comment by DenisNosnitsin1970** [comment]
Daria, I tend to treat them as miniatures by default, unless smth other is indicated. We can always add "cert=", or transcribe what exactly the catalogue says. As to "independent", it was not my idea, I assume it may have been somehow inspired, directly or indirectly, by a definition like this, MINIATURE - "An independent illustration, as opposed to a scene incorporated into another element of the decorative scheme such as a BORDER or INITIAL...)." (https://www.bl.uk/catalogues/illuminatedmanuscripts/GlossM.asp), makes sense, but not so much for the Ethiopian context. I see the points, just to get it right, how would you classify depictions in the images appended, miniatures or drawings? 2 colours but no paints (I meant this type, now I do not find a good example of a narrative picture drawn with two colours, but no paints).
<img width="480" alt="AGM 006 26" src="https://user-images.githubusercontent.com/18533839/129964982-f8e2feb8-1dbc-40ff-9bf8-c6756e43e93c.png">
<img width="615" alt="AGM 006 37" src="https://user-images.githubusercontent.com/18533839/129964985-5e2fa581-5f2e-4127-87cd-653cfc416461.png">

**Comment by DariaElagina** [comment]
Dear @DenisNosnitsin1970 , my classification of those pictures would definitely depend on the system (Guidelines) in which I have to describe them. In the existing system of definitions (BM), these pictures are clearly drawings for me. If we will decide to consider all pictures with at least two colors miniatures, I would encode them as miniatures. That would be a little strange for me, but only because I have another understanding of a miniature based on other experiences. However, if we document that any bichrome or polychrome picture is in the frames of our project treated as a miniature, it would be fine. In the end, it is just some type of convention. (I personally think that they are sigils, and I would put that into desc.)

On the other hand, taking into consideration the existing definition of a miniature in the project, I have also difficulties with the classification of such pictures as miniatures despite them being polychrome:
![IMG_0094-e1510083852594](https://user-images.githubusercontent.com/29276221/130046050-edc30320-4c8a-441a-8804-10da4d12f63b.jpg)

The reason is that in the definition we have miniatures should be "illustrations" and of a special size (An independent illustration in colour that occupies the full-page or part of the equivalent space of one or more columns of text). But this picture, as far as I understand, does not illustrate the text (that is it does not explain, clarify, illuminate, visually represent, or merely decorate), it protects and heals, functions as a "seal" for the protective text, does not allow its power to fade away, and so on... I might overcomplicate things, but at least it is a legitimate way of thinking since the Ethiopic culture ascribes to them this special function. (And again, drawing can also be an illustration, but does not have this aspect in the definition, which is correct in my opinion, but highlights the "illustrative" aspect of a miniature as opposition to a drawing.)

However, if we define a miniature as "any (bichrome or) polychrome image", there would be no problem.
Speaking in the language of mathematic sets, now we have the following situation:
![Set](https://user-images.githubusercontent.com/29276221/130051526-17d60551-3aa1-4bfc-acd9-0186715a99c5.png)
Our definitions, if they are strictly followed, especially that of a miniature, enclose only a part of the images which exist in Ethiopic manuscripts. That means that we have a grey zone, in which everyone decides how to encode this or that image according to one's own judgment, formalists from the formal point of view, functionalists from the functional, and so on...

**Comment by nafisa-valieva** [comment]
@eu-genia @DenisNosnitsin1970 @thea-m, the only place where I could find definitions is https://github.com/BetaMasaheft/Documentation/issues/888 provided mostly by @gnizla, but here in this isssue debated by him :-) 
I did not find any definition of miniature/drawing in the guidelines. If I am to change the definitions proposed https://github.com/BetaMasaheft/Documentation/issues/888 following what  @gnizla proposed in this issue,  it would give:
instead of: Drawing = A monochrome or bichrome image that is not painted
Drawing = A monochrome image that is not painted
instead of: Miniature = An illustration in colour that occupies the full-page or part of the equivalent space of one or more columns of text.
But I need help to understand where I am supposed to change the definition, since I see it only here and in the closed issue.
Alternatively, I just close this issue.

**Comment by eu-genia** [comment]
the issue https://github.com/BetaMasaheft/Documentation/issues/888 (please note the correct link, I do not think your links work) was about definitions for the schema (the prompts that appear in the Oxygen), it is not related to the guidelines

**Comment by nafisa-valieva** [comment]
> the issue [#888](https://github.com/BetaMasaheft/Documentation/issues/888) (please note the correct link, I do not think your links work) was about definitions for the schema (the prompts that appear in the Oxygen), it is not related to the guidelines

Yes, I understand that it was about definitions for the schema. Since Daria referred to the same definitions, this issue is about schema definitions too. Am I supposed to edit the definitions for the schema? Do we need them anywhere in the Guidelines as well?

**Comment by eu-genia** [comment]
(1) Whichever taxonomy and whichever definitions one uses it will always and forever be disputable as people vary and their approaches vary. E.g. I see that in the discussions above Jacopo suggested to suppress illustration as doubling miniature. This is however e.g. not the approach used by Qalamos who distinguish between Illustrations as images related to text (can be miniatures or drawings) and Non-illustrative decoration (which too can be miniatures or drawings but not illustrating the text as far as I understand) https://www.qalamos.net/content/search/search-form-illustration-flexible.xed (used there currently only in Islamic mss cataloguing). 

![Image](https://github.com/user-attachments/assets/fcd03a99-b387-435e-ba9c-b94a361b389f)

So we just need to agree on our understanding and document it clearly.
(2) The definitions in the schema must correspond to what we agree upon, but some principles and examples esp. in cases of doubt would also be helpful in the guidelines, so that one does not have to click through all the options Oxygen gives you in the list to check the definitions but can also have a central lookup as I do not think everyone checks https://betamasaheft.eu/Guidelines/?id=schemaView where one can well read


value | definition
-- | --
frame | an ornament around the page
miniature | An independent illustration in colour that occupies the full-page or part of the equivalent space of one or more columns of text.
band | A border that surrounds a text or image and may occupy the margins and intercolumnar space. Typically used to mark the beginning of a new text or section of text.
punctuation |  
headpiece | A decorative border that surrounds a text and may occupy the margins and intercolumnar space. Typically used to mark the beginning of a new text or section of text. With two specifications, for Band (single border) and Frame (border encompassing multiple sides of text)
headpieceFrame | Decorative frame encompassing all four sides of the text and may occupy the margins and intercolumnar space. Typically used to mark the beginning of a new text or section of text.
headpieceBand | Single decorative border that surrounds a text and may occupy the margins and intercolumnar space. Typically used to mark the beginning of a new text or section of text.
doodles | crude drawings which do not belong to the primary strata of the manuscript (already encoded as “extras”)
drawing | A monochrome or bichrome image that is not painted.
diagram | A drawing that conveys information or knowledge in schematic form.
initial | An enlarged letter at the beginning of a text of paragraph that features figurative or decorative motifs.
map |  
marginal | A small miniature painted next to one of the margins of the text.
mixed |  
other |  
ornamentation |  
paratext |  
printmark |  
publishmark |  
rubrication |  
secondary |  
unspecified |  
unwan |  
engraving | An image on paper created with a printing plate.
printedImage | A printed image, usually of devotional character, glued to a manuscript
overCover | to be used in binding description
overlappingCornerFO | to be used in binding description
overlappingCornerHTO | to be used in binding description
toungedCorner | to be used in binding description
stitchedCorner | to be used in binding description
mitredCorner | to be used in binding description
openMitredCorner | to be used in binding description
Other | to be used in binding description
bindingMaterial | to be used in binding description. Materials of which the binding is made.
Headband | to be used in binding description. Endband at the head end of the spine of a book.
Tailband | for use in binding description. Endband at the tail end of the spine of a book.
Endbands | for use in binding description. Structural and decorative elements of the binding which are found at the head and tail of the spine of a book. The ethiopian endbands are usually slit-braid stripes of leather or link-stitch sewings.
Fastening | for use in binding description. Stripe of different form and material used to keep closed a book. Traces of fastenings can be holes and pins on the outer edge of the boards.
Spine | for use in binding description. The part of the book opposite the fore-edge.
SlipCase | for use in binding description. Portable leather case with attached slip where the manuscript is kept.
Boards | for use in binding description. Components made from one or more pieces of material (usually wood) used to protect and support a bookblock.
SewingStations | for use in binding description. Sewing stations are created by passing a thread through a spine fold when sewing a book.
Sewing | for use in binding description. For the general description of the sewing.
EndLeaves | for use in binding description. All the leaves found at each side of a bookblock, intended to give protection to the text leaves.
Cover | The material that constitutes the immediate covering of the completed binding.


so here we must (1) see the above list if it has to be modified by changing definitions and/or adding types etc (e.g. decorative motifs in magic scrolls can also be defined as "mixed" (then this type can be used for in-between between drawing and miniature if needed) or as "ornamentation" (if it is specified in the definition) - all a matter of our decision) and (2) explain the doubtful cases at https://betamasaheft.eu/Guidelines/?id=decorationDescription.

**Comment by nafisa-valieva** [comment]
I think at this moment nothing needs to be added or modified, until another need for a discussion re-emerge. Daria felt strong about the definitions, I think the current team had no issues with it. 
As for adding cases of doubt to the guidelines, I think it is better to wait until a real case comes. For the moment we did not have doubts, I think. @thea-m @CarstenHoffmannMarburg @MagdaKrzyz @DenisNosnitsin1970, if you all agree, I will close this issue.

**Comment by CarstenHoffmannMarburg** [comment]
I agree with @DariaElagina. To me, a miniature is a small painting that requires special skills and preparation.  In contrast, something that I could produce in five minutes by myself, I used to describe it as 'drawings'. In cases where I am not sure, because information was lacking, I have opted to define it as a 'drawing' because I feel that this is the more general category. For example, I described all the pictures in the Magic Scroll of Strelcyn catalogue in the British Library (1978) as drawings, e.g. for BLorient13226.

```
<decoDesc>
                            <decoNote xml:id="d1" type="miniature">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>
                            <decoNote xml:id="d2" type="miniature">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>
                            <decoNote xml:id="d3" type="miniature">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>    
                            <decoNote xml:id="d4" type="miniature">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>
                        </decoDesc>
```

The definition in the Guidelines could be refined.

**Comment by nafisa-valieva** [comment]
> I agree with [@DariaElagina](https://github.com/DariaElagina). To me, a miniature is a small painting that requires special skills and preparation. In contrast, something that I could produce in five minutes by myself, I used to describe it as 'drawings'. In cases where I am not sure, because information was lacking, I have opted to define it as a 'drawing' because I feel that this is the more general category. For example, I described all the pictures in the Magic Scroll of Strelcyn catalogue in the British Library (1978) as drawings, e.g. for BLorient13226.
> 
> ```
> <decoDesc>
>                             <decoNote xml:id="d1" type="miniature">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>
>                             <decoNote xml:id="d2" type="miniature">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>
>                             <decoNote xml:id="d3" type="miniature">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>    
>                             <decoNote xml:id="d4" type="miniature">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>
>                         </decoDesc>
> ```
> 
> The definition in the Guidelines could be refined.

From what you have posted here, I see that you defined it as type="miniature" in the first place and then as "MagicDrawing". I think it is still in line with the definition. However, to define ´drawing´as something less professional, the difference is really about colours. The Guidlines does not specify how to encode such cases, but give definitions: https://betamasaheft.eu/Guidelines/?q=decoNote&id=decoNote

**Comment by CarstenHoffmannMarburg** [comment]
You are right. According to my own argument, I should better use the following codes here:

```
<decoDesc>
                            <decoNote xml:id="d1" type="drawing">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>
                            <decoNote xml:id="d2" type="drawing">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>
                            <decoNote xml:id="d3" type="drawing">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>    
                            <decoNote xml:id="d4" type="drawing">
                                <desc><term key="MagicDrawing">Magic picture</term>.</desc>
                            </decoNote>
                        </decoDesc>
```
I think, I should rework the decoDesc of each magic scroll, that I have encoded up to now. To my understanding "Magic pictures"/"Magic Drawings" are not miniatures.

For conformity, we should agree on either the one or the other mode.

**Comment by nafisa-valieva** [comment]
> You are right. According to my own argument, I should better use the following codes here:
> 
> ```
> <decoDesc>
>                             <decoNote xml:id="d1" type="drawing">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>
>                             <decoNote xml:id="d2" type="drawing">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>
>                             <decoNote xml:id="d3" type="drawing">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>    
>                             <decoNote xml:id="d4" type="drawing">
>                                 <desc><term key="MagicDrawing">Magic picture</term>.</desc>
>                             </decoNote>
>                         </decoDesc>
> ```
> 
> I think, I should rework the decoDesc of each magic scroll, that I have encoded up to now. To my understanding "Magic pictures"/"Magic Drawings" are not miniatures.
> 
> For conformity, we should agree on either the one or the other mode.

Would it help to expand the definitions in this way: 
miniature: An illustration in colour that occupies the full-page or part of the equivalent space of one or more columns of text. This definition does not apply to magical scrolls, in which all illustrations are defined as drawings. 
drawing: A monochrome or bichrome image that is not painted, or any sort of image in a magic scroll
?

**Comment by DenisNosnitsin1970** [comment]
I wonder if you are troubled by athis detail: "MagicDrawing" would be authomatically associated with the definition of "drawing" (uncoloured depiction), are magic pictures coloured or not?   At least sometimes they are, [https://www.metmuseum.org/art/collection/search/320962].

**Comment by nafisa-valieva** [comment]
> I wonder if you are troubled by athis detail: "MagicDrawing" would be authomatically associated with the definition of "drawing" (uncoloured depiction), are magic pictures coloured or not? At least sometimes they are, [https://www.metmuseum.org/art/collection/search/320962].

that's why I suggest to define drawing as 'A monochrome or bichrome image that is not painted, or any sort of image in a magic scroll'

**Comment by nafisa-valieva** [comment]
> I wonder if you are troubled by athis detail: "MagicDrawing" would be authomatically associated with the definition of "drawing" (uncoloured depiction), are magic pictures coloured or not? At least sometimes they are, [https://www.metmuseum.org/art/collection/search/320962].

are you suggesting to redefine 'MagicDrawing' as any sort of image that appears in a magic scroll? I could not see immediately whether 'MagicDrawing' was already defined.

**Comment by nafisa-valieva** [comment]
@CarstenHoffmannMarburg, I have just noticed that you have self-assigned this issue. Would you like to take over? I think it is more relevant for you than any of us, since you are working with magic scrolls.

**Comment by DenisNosnitsin1970** [comment]
'A monochrome or bichrome image that is not painted, or any sort of image in a magic scroll' - but in this way a contradiction will be introduced into the definition (in codixes drawing is not painted and miniature is painted; but drawing can be painted/not painted in magic scrolls, any sort of) that, I think, one would shoud better avoid. The problem is, in my opinion, a small one: the common understanding of miniatures and drawings is fine, more or less, one should only distinguish the same for the talismanic pictures.

**Comment by CarstenHoffmannMarburg** [comment]
> [@CarstenHoffmannMarburg](https://github.com/CarstenHoffmannMarburg), I have just noticed that you have self-assigned this issue. Would you like to take over? I think it is more relevant for you than any of us, since you are working with magic scrolls.

I am motivated to rework the decoDesc of possibly all manuscript records which contain the type=miniature or type=drawing to achieve more uniformity, as far as this is feasible. At least for the manuscript records, which contain the keyword 'Magic' this should be doable. 

I am also motivated to rework the definitions in the guidelines, if there is agreement on this in this issue. If you agree, I can volunteer  to create a pull request for it.

**Comment by CarstenHoffmannMarburg** [comment]
@eu-genia, I wonder, why I cannot find AuthFile "MagicDrawing", when looking for it in the simple search. Is there a problem with expanding or synchronizing? As far as I see, the file MagicDrawing.xml is on master.

![Image](https://github.com/user-attachments/assets/d5a58e07-5b44-4a06-8c7b-68b63fa84c84)

**Comment by eu-genia** [comment]
did not make it through expand. should be there now

**Comment by CarstenHoffmannMarburg** [comment]
> 'A monochrome or bichrome image that is not painted, or any sort of image in a magic scroll' - but in this way a contradiction will be introduced into the definition (in codixes drawing is not painted and miniature is painted; but drawing can be painted/not painted in magic scrolls, any sort of) that, I think, one would shoud better avoid. The problem is, in my opinion, a small one: the common understanding of miniatures and drawings is fine, more or less, one should only distinguish the same for the talismanic pictures.

I would like to distance myself from the idea of polychromy as the only or most decisive criteria for the distinction between miniature and drawing as @gnizla has advocated for. To my understanding, the principal point is the craftmanship, precision and preparation, that is considerably higher for miniatures as it is for drawings. The presence of many colours is one aspect of preparation, precision and craftmanship. For a simple drawing, the executor would take the colours, that he has at hand, especially those, that are used for writing the text or for ornamental bands. For a miniature, a specific set of colours, which are necessary will be procured, produced and specially prepared for the painting process. The multicolored image, which @DariaElagina posted above, according to this definition falls into the category of a drawing, because even though. there are four colours involved (black, red, yellow, violet), these are the ordinary set of colours, which are often used for ornamental bands or similar ornamentation and which every scribe would have at hand without special procurement, as it is the case in the following picture. As a consequence, I would define a drawing as "**a picture executed with simple means without special procurement or selection of colours, specialisation or preparation**". Miniatures should be therefore defined as "**Finely executed polychrome painting that requires a certain skill, special training and meticulous preparation**".

![Image](https://github.com/user-attachments/assets/cbca1e19-966a-4966-b6ea-6b5438d51f0a)

However, I am not an art historian. Maybe @Ilaria9515 @GueshSol @gnizla will disagree to that definition of miniature and drawing.

**Comment by nafisa-valieva** [comment]
@CarstenHoffmannMarburg, I think you are going too far away, and it is not helpful from my point of view, because the definition you suggest is judgmental, whereas it should not be. @DenisNosnitsin1970 has suggested three times in this issue to introduce a category apart - talismanic pictures. Although Daria and Jacopo rejected them, no one offered a better solution. I suggest we discuss whether we want this extra category. I do not have any objections; on the contrary, given that no better solution is found, it seems to be the only solution at hand. I suggest we vote! @eu-genia @eu-genia @MagdaKrzyz @thea-m, are you objecting to ´talismanic pictures' as a category apart? 
(no voting, because Eugenia noted below that  a category apart will make the project´s encoding incompatible with other projects)

**Comment by eu-genia** [comment]
I would suggest that we always try to align to existing taxonomies and definitions and not invent our own especially in fields where we are not specialists, unlike Jacopo, who as art historian deals with manuscript decoration on a daily basis (and we see that even for him definitions are difficult).

I do not think that the skill is what distinguishes a drawing from a painting/miniature and I would strongly distance myself from such definition. Leonardo da Vinci left paintings and drawings, all of them are skillful. The difference is the technique used.

If you want to study art history you may want to start with some handbook containing definitions like Brown's Understanding Illuminated Manuscripts: A Guide to Technical Terms (I have uploaded it for you to https://drive.google.com/file/d/149cHxrnzqSpYtz0fb5DBsip7PSTkcsSX/view?usp=sharing) - of course it is meant for western manuscripts but still - it additionally distinguishes between outline drawings and tinted drawings. 

In the discussion started by Daria I would still sustain the view that elaborate painted figures in magic manuscripts are miniatures=paintings as opposed to drawings=mostly outlines, whether filled or not with colour.

**Comment by eu-genia** [comment]
I am against talismanic pictures, the main typology should still be miniature/drawing/ornamentation/frame. Content matter should be encoded in keywords and art themes. Ideally we can very precisely categorize each image with Iconclass (that must be there for art themes but often is not provided).  We should always think in universal ontologies/taxonomies and not invent creative encoding that will make us incompatible with others.

**Comment by nafisa-valieva** [comment]
@eu-genia, do I understand you correctly: you do not want to modify the existing definition and do not want to have a category ´talismanic pictures´? I am fine with this too, because I did not have problems with my work. @CarstenHoffmannMarburg, do you think you can continue using the existing definitions and apply them as suggested by Eugenia to the miniatures in the magic scrolls?

**Comment by eu-genia** [comment]
Summing up, I think our definition of drawing (NOT PAINTED) is fully correct. Painted images are miniatures even if they are abstract.

**Comment by CarstenHoffmannMarburg** [comment]
> [@CarstenHoffmannMarburg](https://github.com/CarstenHoffmannMarburg), I think you are going too far away, and it is not helpful from my point of view, because the definition you suggest is judgmental, whereas it should not be. [@DenisNosnitsin1970](https://github.com/DenisNosnitsin1970) has suggested three times in this issue to introduce a category apart - talismanic pictures. Although Daria and Jacopo rejected them, no one offered a better solution. I suggest we discuss whether we want this extra category. I do not have any objections; on the contrary, given that no better solution is found, it seems to be the only solution at hand. I suggest we vote! [@eu-genia](https://github.com/eu-genia) [@eu-genia](https://github.com/eu-genia) [@MagdaKrzyz](https://github.com/MagdaKrzyz) [@thea-m](https://github.com/thea-m), are you objecting to ´talismanic pictures' as a category apart?

I would not support the idea of creating a new category called 'talismanic'. Firstly, the word 'talismanic' is problematic and imprecise. Secondly, we already have the 'authFile' 'MagicDrawing', which was created to define all pictures designed and used for magical purposes.  Creating a new category would require reworking all existing records containing this tag. I don't know if Denis suggested it before MagicDrawing was created, but either way, we don't need it anymore.

I am not sure if voting by majority is a useful way to make this decision. In the past, we either kept everything as it was, or we asked @abausi to make a decision. I am open to implementing any solution, but I still believe that a distinction based solely or primarily on the number of colours involved is superficial and goes against the intuition of researchers.

**Comment by eu-genia** [comment]
@CarstenHoffmannMarburg please read the manual I uploaded before considering introducing new definitions in the field of manuscript decoration, thank you. 

This is undoubtedly a drawing 

![Image](https://github.com/user-attachments/assets/f72125c4-7042-4c0d-b3b3-cb0a62ab2f77)

I dare say the level of skill and preparation involved here is somewhat more than in the miniatures posted above.


I suggest we close the issue.

**Comment by nafisa-valieva** [comment]
@CarstenHoffmannMarburg, I agree to closing the issue. Do you think what you have encoded thus far is in line with the existing definitions? If not, maybe just bring them in line with definitions, even if it goes against your intuition?

**Comment by CarstenHoffmannMarburg** [comment]
> [@CarstenHoffmannMarburg](https://github.com/CarstenHoffmannMarburg), do you think you can continue using the existing definitions and apply them as suggested by Eugenia to the miniatures in the magic scrolls?

Of course, I am happy with whatever decision is made. Assuming that the concept described in the guidelines remains unchanged, I also assume that I do not need to rework existing manuscript records such as BLorient13226, which I mentioned above.

**Comment by nafisa-valieva** [comment]
> > [@CarstenHoffmannMarburg](https://github.com/CarstenHoffmannMarburg), do you think you can continue using the existing definitions and apply them as suggested by Eugenia to the miniatures in the magic scrolls?
> 
> Of course, I am happy with whatever decision is made. Assuming that the concept described in the guidelines remains unchanged, I also assume that I do not need to rework existing manuscript records such as BLorient13226, which I mentioned above.

Perfect!
